res_syntax/tests/oprint/expected/oprint.resi.txt

```diff
@@ -486,4 +486,4 @@ type emptyObject = {.}
 let f: (~x: 'a=?, ~y: 'b) => option<'a>
 type call = CleanStart
 let f: (~a: int=?, unit) => int
-type opt = {x: int, y?: string}
\ No newline at end of file
+type opt = {x: int, y?: option<string>}
```

Comment:
Need to check what outcome printer is now used.

Reply:
OK this is good. The type checker in the compiler libs shipped with the syntax had fallen behind.